ui/pages/confirmation/components/confirmation-network-switch/index.scss

```diff
@@ -0,0 +1,56 @@
+.confirmation-network-switch {
+  display: flex;
+  height: 100%;
+  justify-content: center;
+
+  &__result {
+    @include H3;
+
+    position: absolute;
+    top: 50%;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    text-align: center;
+    color: var(--Black-100);
+  }
+
+  &__icons {
+    display: flex;
+  }
+
+  &__icon {
+    display: block;
```

Comment:
display block can be achieved on the Box component

Reply:
Done.